.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @sil-org/php-devs
```

Comment:
Random thought: it would be nice if they would have a similar validity check for workflow files.